### DISCUSSION TITLE
Arrangement of calendar icon display

### DISCUSSION THREAD

**Discussion by davidovic999** [mikeberger/borg_calendar#209] [Ideas]
Hi Mike,
For easy, intuitive access to calendars, it's best to arrange calendar icons in ascending order: day, week, month, year.

![calendars-1](https://github.com/mikeberger/borg_calendar/assets/59475468/af6dc594-8dba-425d-8cb2-e02cde18b880)

![calendars-2](https://github.com/mikeberger/borg_calendar/assets/59475468/1eb5106d-9490-4be9-8deb-fd8e589c3e7b)

Thanks and have a great day,

**Comment by mikeberger** [comment]
A trivial change. they are currently in the order they were added to the program.

**Comment by davidovic999** [comment]
It was simply a matter of improving the UI experience. Don't worry, the program is perfect as is. It does its job.

**Comment by mikeberger** [comment]
It was a trivial change...meaning that I made the code change that you suggested and committed it before responding. 

https://github.com/mikeberger/borg_calendar/commit/8eb4f04ad3416d5f1775d1c007b1c6a20f62a3f3
